reference/commands/consumer/install.rst

```diff
@@ -312,6 +322,24 @@ With the :command:`-o` parameters you can only define specific package options.
     You can use :ref:`profiles <profiles>` files to create predefined sets of **settings**,
     **options** and **environment variables**.
 
+conf
+----
+
+With the :command:`-c` parameters you can define specific package configurations.
+
+.. code-block:: bash
+
+    $ conan install . -c tools.microsoft.msbuild:verbosity=Diagnostic
+    $ conan install . -c core:required_conan_version>=1.34 -c tools.build:processes=10
```

Comment:
The ``core`` configuration does not work that way, because it is checked elsewhere in the flow, it is not obtained from the profile.

I'd recommend to document things from the tests of the feature, wherever possible, this is wrongly assuming this will work.

Reply:
Absolutely right. Fixed!